### PR TITLE
KO: docs/site/ko/guide/checkpoint.ipynb

### PR DESCRIPTION
**Pull request by paulkim1997**
translated checkpoint.ipynb
EN -> KO

EN: [https://github.com/tensorflow/docs/blob/master/site/en/guide/checkpoint.ipynb](url)

### CODE REVIEW

**Comment by paulkim1997** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdocs%2Fpull%2F1186) for more info**.

@googlebot I signed it!

File: site/ko/guide/checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,790 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "3TI3Q3XBesaS"
+      },
+      "source": [
+        "# Checkpoint 훈련하기"
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: I suggest 'Checkpoint' -> '체크포인트'.
http://bit.ly/tf-docs-translation-glossary

File: site/ko/guide/checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,790 @@
...
+        "\n",
+        "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 공식 영문 문서의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 tensorflow/docs 깃헙 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 참여하려면 docs-ko@tensorflow.org로 메일을 보내주시기 바랍니다.",
+        "\"텐서플로 모델 저장하기\" 라는 문구는 보통 둘중 하나를 의미합니다:\n",
+        "\n",
+        "  1. Checkpoints, 혹은 \n",
+        "  2. SavedModel.\n",
+        "\n",
+        "Checkpoint는 모델이 사용한 모든 매개변수(`tf.Variable` 객체들)의 정확한 값을 캡처합니다. Chekcpoint는 모델에 의해 정의된 연산에 대한 설명을 포함하지 않으므로 일반적으로 저장된 매개변수 값을 사용할 소스 코드를 사용할 수 있을 때만 유용합니다.\n",
+        "\n",
+        "반면 SavedModel 형식은 매개변수 값(체크포인트) 외에 모델에 의해 정의된 연산에 대한 일련화된 설명을 포함합니다. 이 형식의 Model은 Model을 만든 소스 코드와 독립적입니다. 따라서 TensorFlow Serving, TensorFlow Lite, TensorFlow.js 또는 다른 프로그래밍 언어(C, C++, Java, Go, Rust, C# 등. TensorFlow APIs)들을 통해 배포하기에 적합합니다.\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: For consistency, I suggest 'Model' -> '모델'.
In my opinion, '~들을 통해 배포하기' is unnatural.
I suggest '~들을 통해 배포하기' -> '~로 배포하기'.
How about you?

File: site/ko/guide/checkpoint.ipynb [hunk 3]
```diff
@@ -0,0 +1,790 @@
...
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "VEvpMYAKsC4z"
+      },
+      "outputs": [],
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "try:\n",
+        "  # %tensorflow_version은 오직 Colab에서만 사용가능합니다.\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: 'Colab' -> '코랩'
http://bit.ly/tf-docs-translation-glossary

File: site/ko/guide/checkpoint.ipynb [hunk 4]
```diff
@@ -0,0 +1,790 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "1jpZPz76ZP3K"
+      },
+      "source": [
+        "텐서플로 model의 지속적인 상태는 `tf.Variable` 객체에 저장되어 있습니다. 이들은 직접적으로 구성될 수 있지만, `tf.keras.layers` 혹은 `tf.keras.Model`와 같은 상위레벨 API들로 만들어 지기도 합니다.\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: In my opinion, '직접적으로 구성될' is unnatural.
How about '직접 구성할'?

I suggest '상위레벨' -> '고수준'.
http://bit.ly/tf-docs-translation-glossary

File: site/ko/guide/checkpoint.ipynb [hunk 5]
```diff
@@ -0,0 +1,790 @@
...
+        "\n",
+        "변수를 관리하는 가장 쉬운 방법은 Python 객체에 변수를 연결한 다음 해당 객체를 참조하는 것입니다. \n",
+        "\n",
+        "`tf.train.Checkpoint`, `tf.keras.layers.Layer`, and `tf.keras.Model`의 하위class들은 해당 속성에 할당된 변수를 자동 추적합니다. 다음 예시는 간단한 선형 model을 구성하고, 모든 model 변수의 값을 포합하는 checkpoint를 씁니다."
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: '하위class' -> '하위 클래스'

File: site/ko/guide/checkpoint.ipynb [hunk 6]
```diff
@@ -0,0 +1,790 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "FHTJ1JzxCi8a"
+      },
+      "source": [
+        "### 인위적으로 Checkpoint작성하기"
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: In my opinion, '인위적으로' -> '직접' or '손수' is more natural.

File: site/ko/guide/checkpoint.ipynb [hunk 7]
```diff
@@ -0,0 +1,790 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "DR2wQc9x6b3X"
+      },
+      "source": [
+        "\u003ca id=\"loading_mechanics\"/\u003e\n",
+        "\n",
+        "## 작동 원리\n",
+        "\n",
+        "텐서프로우는 로드되는 객체에서 시작하여 명명된 엣지가 있는 방향 그래프를 통과시켜 변수를 checkpoint된 값과 일치시킵니다. 엣지의 이름들은 특히 기여한 객체의 이름에서 따왔습니다. 예를들면, `self.l1 = tf.keras.layers.Dense(5)`안의 `\"l1\"`. `tf.train.Checkpoint` 이것의 키워드 전달인자 이름을 사용했습니다, 여기에서는 `\"step\"` in `tf.train.Checkpoint(step=...)`.\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: '텐서프로우' -> '텐서플로'

File: site/ko/guide/checkpoint.ipynb [hunk 8]
```diff
@@ -0,0 +1,790 @@
...
+        "\n",
+        "위의 예에서 나온 종속성 그래프는 다음과 같습니다.:\n",
+        "\n",
+        "![Visualization of the dependency graph for the example training loop](http://tensorflow.org/images/guide/whole_checkpoint.svg)\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: 'Visualization ~ loop' is untranslated.

File: site/ko/guide/checkpoint.ipynb [hunk 9]
```diff
@@ -0,0 +1,790 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GqEW-_pJDAnE"
+      },
+      "source": [
+        "이 새로운 개체에 대한 의존도 그래프는 우리가 위에 적은 더 큰 checkpoint보다 작은 하위 그래프입니다. 이것은 오직  `tf.train.Checkpoint`에서 checkpoints 셀때 bias와 저장 카운터만 포함합니다.\n",
+        "\n",
+        "![Visualization of a subgraph for the bias variable](http://tensorflow.org/images/guide/partial_checkpoint.svg)\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: 'Visualization ~ variable' is untranslated.

File: site/ko/guide/checkpoint.ipynb [hunk 10]
```diff
@@ -0,0 +1,790 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GqEW-_pJDAnE"
+      },
+      "source": [
+        "이 새로운 개체에 대한 의존도 그래프는 우리가 위에 적은 더 큰 checkpoint보다 작은 하위 그래프입니다. 이것은 오직  `tf.train.Checkpoint`에서 checkpoints 셀때 bias와 저장 카운터만 포함합니다.\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: 'bias' -> '편향'
http://bit.ly/tf-docs-translation-glossary

File: site/ko/guide/checkpoint.ipynb [hunk 11]
```diff
@@ -0,0 +1,790 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "5fxk_BnZ4W1b"
+      },
+      "source": [
+        "### 목록 및 사전 추적\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: In python, dictionary means '딕셔너리형'.
So, I suggest '사전' -> '딕셔너리'
http://bit.ly/tf-docs-translation-glossary

File: site/ko/guide/checkpoint.ipynb [hunk 12]
```diff
@@ -0,0 +1,790 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "zGG1tOM0L6iM"
+      },
+      "source": [
+        "##Estimator를 사용하여 객체 기반 checkpoint를 저장하기\n",
+        "\n",
+        " [Estimator guide](https://www.tensorflow.org/guide/estimator)를 보십시오.\n",
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: 'Estimator guide' is untranslated.

File: site/ko/guide/checkpoint.ipynb [hunk 13]
```diff
@@ -0,0 +1,790 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "UTKvbxHcI3T2"
+      },
+      "source": [
+        "당신은 래퍼 객체를 목록과 사전에 있음을 알아차릴겁니다. 이러한 래퍼는 기본 데이터 구조의 checkpoint 가능한 버전입니다. 속성 기반 로딩과 마찬가지로, 이러한 래퍼들은 변수의 값이 용기에 추가되는 즉시 복원됩니다. "
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: For readability, I suggest '래퍼 객체' -> '래퍼(wrapper) 객체'.

File: site/ko/guide/checkpoint.ipynb [hunk 14]
```diff
@@ -0,0 +1,790 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "GqEW-_pJDAnE"
+      },
+      "source": [
+        "이 새로운 개체에 대한 의존도 그래프는 우리가 위에 적은 더 큰 checkpoint보다 작은 하위 그래프입니다. 이것은 오직  `tf.train.Checkpoint`에서 checkpoints 셀때 bias와 저장 카운터만 포함합니다.\n",
+        "\n",
+        "![Visualization of a subgraph for the bias variable](http://tensorflow.org/images/guide/partial_checkpoint.svg)\n",
+        "\n",
+        "`restore()` 최적의 어써션을 가진 개체의 상태를 반환해줍니다.  새로 만든 checkpoint에서 우리가 만든 모든 개체가 복원되어 status.assert_existing_objects_match()가 통과합니다."
```
Comment (JKIsaacLee) [inline site/ko/guide/checkpoint.ipynb]: I suggest '`restore()` 최적의 어써션을 가진 개체의 상태를 반환해줍니다.' -> '`restore()` 함수는 최적의 어써션을 가진 개체의 상태를 반환합니다'

File: site/ko/guide/checkpoint.ipynb (hunk 2 above)
Comment (paulkim1997) [reply site/ko/guide/checkpoint.ipynb]: I agree with you statement!

**Comment by paulkim1997** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdocs%2Fpull%2F1186) for more info**.

@googlebot I signed it

**Comment by paulkim1997** [comment]
> I simply reviewed this translation.
> If you have any other suggestion, comment plz. 👍

Thank you for your review!
I've fixed them according to your comments

**Pull request merged**